RTC beta (#1778)
tinymce/tinymce-docs: _includes/auth/jwt-provider-endpoint-url.md

## JWT provider endpoint URL

> The easiest way to setup JWT authentication against {{site.cloudname}} services is to create a JWT provider endpoint. This endpoint takes a JSON HTTP POST request and produces a JSON result with the token that the service will then use for all the HTTP requests.

Reviewer: ```suggestion
The easiest way to setup JWT authentication for {{site.cloudname}} services is to create a JWT provider endpoint. This endpoint takes a JSON HTTP POST request and produces a JSON result with the token that the service can then use for all HTTP requests.
```
maybe? @lnewson please check?

Reviewer 2: `JSON response` maybe instead of `result`? LGTM otherwise though.

Reviewer: updated my suggestion
